### DISCUSSION TITLE
Welcome to projectmate Discussions!

### DISCUSSION THREAD

**Discussion by rohitdasu** [rohitdasu/projectmate#40] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SanDiego2049** [comment]
I feel the homepage's content is too little to be showing a vertical scroll. I don't know what issue that falls under so i couldnt create an issue for it.

**Reply by rohitdasu** [reply]
yeah currently we have less content. We are brainstorming ideas of what can be added to it.

**Reply by SanDiego2049** [reply]
Oh okay. If tthat's the case, I'd suggest the homepage be about educating people on open source. The resources to get started with on github, the advantages of github and then leading them to do their first contributions to build confidence, At the footer of the page, "Not a first time contributor? Check out the projects page" could be placed to link to the advanced projects. The projects page should also have a filter for beginners, intermediates and experts levels. That's all that came off the top of my head right now, i hope i didnt overwhelm you

**Reply by rohitdasu** [reply]
do you have any blueprints for this? i mean designs. It would be great

**Reply by leewan-09** [reply]
It would be nice if you can provide some ideas

**Reply by SanDiego2049** [reply]
Unfortunately, i do not have any. I just came up with hte ideas on the spot, i hadn't given much thought to it's layout.
